.github/workflows/package.yml

```diff
@@ -132,7 +132,13 @@ jobs:
           # - pgroonga-fedora-33
     env:
       APACHE_ARROW_REPOSITORY: ${{ github.workspace }}/apache-arrow
-    runs-on: ubuntu-latest
+    # condition && true-case || false-case
+    # ==
+    # condition ? true-case : false-case
+    runs-on: >-
+      ${{ (contains(matrix.package-id, 'arm64') ||
+           contains(matrix.package-id, 'aarch64')) && 'ubuntu-24.04-arm' ||
+                                                      'ubuntu-latest' }}
```

Comment:
How about adding the comment when we can remove this workaround?

Reply:
This is not a workaround.

Reply:
I'm sorry I misunderstood it. I got it.
ref: https://github.blog/changelog/2025-01-16-linux-arm64-hosted-runners-now-available-for-free-in-public-repositories-public-preview/